addons/medical_gui/CfgUIGrids.hpp

```diff
@@ -0,0 +1,30 @@
+class CfgUIGrids {
+    class IGUI {
+        class Presets {
+            class Arma3 {
+                class Variables {
+                    GVAR(patientInfo)[] = {
+                        {
+                            PATIENT_INFO_IGUI_BASE_X,
+                            PATIENT_INFO_IGUI_BASE_Y,
+                            POS_W(9),
+                            POS_H(20)
+                        },
+                        GUI_GRID_W,
+                        GUI_GRID_H
+                    };
+                };
+            };
+        };
+
+        class Variables {
+            class GVAR(patientInfo) {
+                displayName = COMPONENT_NAME;
+                description = CSTRING(IGUI_PatientInfo);
+                preview = QPATHTOF(ui\patient_info_preview_ca.paa);
+                saveToProfile[] = {0, 1, 2, 3};
```

Comment:
Only the X and Y are used.

Reply:
I've tested it and no matter what I put there all of the coords are saved, also I've checked BI script and there was no reference to `saveToProfile` whatsoever. Unless I was looking in wrong place. I can change it tho.

Reply:
Yeah, that's what I meant. `RscDisplayOptionsLayout` currently does not use the `saveToProfile` entry but all of BI's `CfgUIGrids` classes have it defined as if it did. Should change in case it is ever used in the future.